src/Components/ADT3DSceneBuilder/Internal/Shared/MakePivotRequired.tsx

```diff
@@ -0,0 +1,22 @@
+import { IPivotItemProps } from '@fluentui/react';
+import React from 'react';
+import { customPivotItemStyles } from '../Behaviors/BehaviorsForm.styles';
+
+export function setPivotToRequired(
+    isValid: boolean | undefined,
+    link?: IPivotItemProps,
```

Comment:
Could you change the name of this argument to linkProps? To clarify its purpose.

Reply:
that's the naming convention for the customRendering function for Pivot components. Just wanted to stick to that so it isn't confusing for the next person